src/js/reducers.js

```diff
@@ -0,0 +1,14 @@
+var store = Redux.createStore(bartData);
+
+function bartData(state = { apiData: {} }, action) {
+  switch (action.type) {
+    case "BART_DATA_UPDATE":
+      if (state.apiData === undefined) {
+        return state;
+      }
+      state.apiData = action.apiData;
```

Comment:
This should be in a function

Reply:
state is immutable, you should not update it in this way

Reply:
@eladg Moved to a new file, take a look.